test/article.spec.js

```diff
@@ -153,3 +159,121 @@ describe('ARTICLES', () => {
     });
   });
 });
+
+describe('/PUT articles id', () => {
+  before((done) => {
+    chai
+      .request(app)
+      .post('/api/v1/user/login')
+      .send(user2)
+      .end((err, res) => {
+        if (!err) {
+          userToken = res.body.token;
+        }
+        done();
+      });
+  });
+
+
+  it('should update an article', (done) => {
+    chai
+      .request(app)
+      .put(`/api/v1/articles/${articleId}`)
+      .set('authorization', userToken)
+      .send(article2)
+      .end((err, res) => {
+        expect(res).to.have.status(200);
+        expect(res.body).to.have.property('success').equal(true);
+        expect(res.body).to.have.property('message').equal('Article updated successfully');
+        done(err);
+      });
+  });
+
+  it('should return an error if the article is not found', (done) => {
+    chai
+      .request(app)
+      .put('/api/v1/articles/eab6fbb6-aeda-4e1b-b4be-3582f51a6d30')
+      .set('authorization', userToken)
+      .send(article2)
+      .end((err, res) => {
+        expect(res).to.have.status(404);
+        expect(res.body).to.have.property('success').equal(false);
+        expect(res.body.errors).to.be.an('Array');
+        expect(res.body.errors[0]).to.be.equal('Article not found');
+        done(err);
+      });
+  });
+});
+
+describe('/PUT articles id', () => {
+  before((done) => {
+    chai
+      .request(app)
+      .post('/api/v1/user/')
+      .send(myUser)
+      .end((err, res) => {
+        userToken2 = res.body.token;
+        done();
+      });
+  });
+
+  before(() => { updateVerifiedStatus(myUser.email); });
+  it('should return an error if the user is not the owner of the article', (done) => {
+    chai
+      .request(app)
+      .put(`/api/v1/articles/${articleId}`)
+      .set('authorization', userToken2)
+      .send(article2)
+      .end((err, res) => {
+        expect(res).to.have.status(401);
+        expect(res.body).to.have.property('success').equal(false);
+        expect(res.body.errors).to.be.an('Array');
+        expect(res.body.errors[0]).to.be.equal('You are unauthorized to perform this action');
+        done(err);
+      });
+  });
+});
+
+describe('/DELETE articles id', () => {
+  it('should return an error if the user is not the owner of the article', (done) => {
+    chai
+      .request(app)
+      .delete(`/api/v1/articles/${articleId}`)
+      .set('authorization', userToken2)
+      .send(article2)
```

Comment:
Not sure you need to send an article when deleting.

Reply:
True. Thanks

Reply:
Done